app/presenters/nsm/tasks/case_outcome.rb

```diff
@@ -0,0 +1,12 @@
+module Nsm
```

Comment:
not needed in this PR since it's handled in: https://dsdmoj.atlassian.net/browse/CRM457-2288

but welcome to keep in this ticket as 2288 is a big ticket and anything we can do to shrink it down is welcome

Reply:
Yeah I was just trying to 1:1 implement the functionality, I realised now when the task tests failed.

If there's any knock-off to having the here it's an easy removal.